src/backend/marsha/core/tasks/timed_text_track.py

```diff
@@ -62,7 +62,7 @@ def convert_timed_text_track(timed_text_track_pk, stamp):
         prefix_destination = timed_text_track.get_videos_storage_prefix(stamp)
 
         with video_storage.open(source, "rt") as timed_text_file:
-            timed_text = timed_text_file.read()
+            timed_text = timed_text_file.read().replace("\ufeff", "")
```

Comment:
It looks like we could also open the file with `encoding="utf-8-sig"`, which would strip the BOM.
Both are fine to me!

Reply:
Oh good point, that's a shame